modules/37_industry/subsectors/bounds.gms

```diff
@@ -21,7 +21,7 @@ vm_cesIO.up(ttot,regi,"ue_steel_secondary")
       p37_cesIO_up_steel_secondary(ttot,regi,"%cm_GDPscen%")
     );
 $elseif.secondary_steel_bound "%cm_secondary_steel_bound%" == "scenario"
-if (1 eq cm_emiscen,
+if (cm_startyear eq 2005,
```

Comment:
These changes are not equivalent.  Not all policy scenarios have to start later then 2005.  Not all scenarios that are fixed after 2005 are baseline scenarios.

Reply:
right @robertpietzcker, it is more the other way around: from now on the first scenarios of a cascade will not be baselines anymore but can be a NPi scenario

Reply:
right, my question was meant to be addressed to @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q to make sure this is a piece of code he doesn't want executed in REMIND anymore (or only for very specific "how would a baseline look like" experiments)

Reply:
https://github.com/remindmodel/remind/blob/af701bda24f0e0c1eafe5281a3fb299ebad2c4a8/modules/37_industry/subsectors/bounds.gms#L24-L28

If `NPi` is to be the new baseline and calibrated to `mrremind` output, then this upper bound should apply to it.

> I think the reason for this replacement was that in many places in the code, the "emiscen = 1" was used to say "the first REMIND run in a cascade", so it was widely replaced throughout the code.

Well, in this case, `cm_emiscen = 1` was used to say "if this is not a policy scenario".

Reply:
ok, based on your first sentence the pull request is indeed right, as it makes NPi the scenario that is calibrated to the mrremind output

Reply:
> ok, based on your first sentence the pull request is indeed right

Incidentally so.  `cm_startyear eq 2005` is not the same as "if this is not a policy scenario".  I would suggest to avoid the
>  in many places in the code, the _"X"_ was used to say _"Y"_

situation and be explicit about what is meant.

Making `NPi` the "baseline" and `Base` a counter-factual renders `cm_emiscen` unsuited for this task.  But then we should have an explicit parameter to distinguish whether something is a baseline or climate policy scenario and which assumptions and parameters should apply to it.

Reply:
"cm_calibrationRun"? "cm_firstRunInCascade"? "cm_takesFEpathsFromMrremind"? @LaviniaBaumstark

Reply:
"cm_firstRunInCascade" fits best what we mean with it as it is not only relevant for calibration scenarios

Reply:
but isn't this also the same as start_year = 2005 ?

Reply:
regarding adding a new switch called "cm_firstRunInCascade": if really necessary, sure; if not I would like to avoid adding a new switch would need to be added to each scenario_config file